index.js

```diff
@@ -1,17 +1,49 @@
 class SortedList {
-  constructor() {}
+  constructor(items, length) {
+    this.items = [];
+    this.length = 0;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort((a, b) => a - b);
+  }
 
-  get(pos) {}
+  get(pos) {
+    if ((pos = this.items.length - 1)) {
```

Comment:
Estas seguro que ese `if` funciona? 
me parece que lo que querias hacer es `pos === this.items.length - 1`
igualmente te recomiendo podes si no hacer `!this.items[pos]`, por que con tu solucion, si te pasan una pos mayor a `this.items.length - 1` no entraria en el error